packages/trader/src/Stores/Modules/Contract/Helpers/chart-notifications.js

```diff
@@ -0,0 +1,14 @@
+import React        from 'react';
+import { localize } from 'App/i18n';
+import Localize     from '../../../../App/Components/Elements/localize.jsx';
+
+export const switchToTickChart = () => ({
+    key    : 'switch_to_tick_chart',
```

Comment:
can this be assign to the object itself? rather than calling function to return the object?

Reply:
sure, was just following the convention when I exported it out. 😄